src/hotspot/cpu/s390/interp_masm_s390.cpp

```diff
@@ -1077,7 +1077,11 @@ void InterpreterMacroAssembler::lock_object(Register monitor, Register object) {
 void InterpreterMacroAssembler::unlock_object(Register monitor, Register object) {
 
   if (UseHeavyMonitors) {
```

Comment:
In our understanding, call_VM_leaf means this function will not break safepoint and call Java functions. But in WispMonitor, we will call Java functions to do unpark. Therefor we need to use call_VM instead of call_VM_leaf when using WispMonitor. So we change all the call_VM_leaf with InterpreterRuntime::monitorexit to call_VM with InterpreterRuntime::monitorexit_wisp for all platform.